cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -126,6 +127,23 @@ message DisplayVariable {
   // Optional.
   // Gen: manually-authored
   string boolean_group = 20; // @gotags: json:"booleanGroup,omitempty" yaml:"booleanGroup,omitempty"
+
+  // Alternate default value.
+  // This allows authors to define an alternative value for pre identified usecases such as security.
```

Comment:
Could you add some more context here? "AlternateDefault" name gives an impression that this field will provide a different value in place of the _default_value_ field in _BlueprintVariable_(bpmetadata.proto), is that correct? Is it used for providing supporting information for the default_value?

Reply:
Added some more info

> "AlternateDefault" name gives an impression that this field will provide a different value in place of the default_value field in BlueprintVariable(bpmetadata.proto), is that correct?

Yup! Its intended to be a hand authored alternate to the one in BlueprintVariable.